tezos/messages/src/protocol/proto_006/contract.rs

```diff
@@ -30,3 +30,22 @@ has_encoding!(Counter, COUNTER_ENCODING, {
             Field::new("counter", Encoding::Z)
         ])
 });
+
+#[derive(Serialize, Deserialize, Debug, Clone, Getters)]
+pub struct Balance {
+    #[get = "pub"]
+    balance: BigInt,
+}
+
+impl Balance {
+    pub fn to_string(&self) -> String {
```

Comment:
@adonagy 
this is againts clippy: https://rust-lang.github.io/rust-clippy/master/#inherent_to_string
I would rename that function to something like: `to_json_format`,
and I also see another usage here: `impl Counter {` 

and also please check other usage `fn to_string` and rename them specificcally

Reply:
Oh right, I had a hunch this was not entirely ok. Will do 👍